lib/map-store.js

```diff
@@ -96,6 +96,13 @@ MapStore.prototype.transformCoverage = function (coverageMap) {
         return fs.readFileSync(pathutils.asAbsolute(filePath, that.baseDir));
     };
 
+    coverageMap.files().forEach(function (file) {
```

Comment:
@DatenMetzgerX sorry for taking so long to land this, I was hoping someone would chime-in to help with the review who has a more source-map-focused workflow.

Would it be possible to add a test for this new logic that fails until we land the changes in istanbul-lib-instrument.